utils/release/git-publish-all.mjs

```diff
@@ -146,6 +154,13 @@ const getCliChangelog = () => {
   });
 })();
 
+/**
```

Comment:
It's mostly to 'type' the param of the function, e.g. in typescript it would have been:
```ts
async function commitChanges(releaseNumber: string|number, commitMessage: string, octokit: Octokit) {
```

Reply:
Yes a brief desc would suffice - it would also make it look less weird / incomplete.